src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp

```diff
@@ -745,11 +745,31 @@ class npc_a_special_surprise : public CreatureScript
     };
 };
 
+enum DevourHumanoid
+{
+    SPELL_DEVOUR_HUMANOID = 53111
+};
+class spell_death_knight_devour_humanoid : public SpellScript
+{
+    PrepareSpellScript(spell_death_knight_devour_humanoid);
+
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
```

Comment:
Extra spaces

Reply:
From what I saw these spaces are kept in TC.

Reply:
Both the `/* effIndex */` and the `/*effIndex*/`variants can be found in current source.
`/*effIndex*/` wins 419 - 47, though.